java/src/main/java/ai/rapids/cudf/ParquetWriterOptions.java

```diff
@@ -73,12 +94,20 @@ private ParquetWriterOptions(Builder builder) {
     super(builder);
     this.statsGranularity = builder.statsGranularity;
     this.isTimestampTypeInt96 = builder.isTimestampTypeInt96;
+    this.precisions = builder.precisionValues;
   }
 
   public StatisticsFrequency getStatisticsFrequency() {
     return statsGranularity;
   }
 
+  /**
+   * Return the flattened list of precisions if set otherwise empty array will be returned.
```

Comment:
The docs here need an explanation of flattened, either directly here or pointing to the example in `withPrecisionValues`.